Source/Lib/Codec/EbDefinitions.h

```diff
@@ -668,6 +668,28 @@ extern    EB_U32                   libMutexCount;
     } \
     libMutexCount++;
 
+#define EB_STRDUP(dst, src) \
+    dst = (char*) malloc(strlen(src)+1); \
+    strcpy_ss((char*)dst, strlen(src)+1, src);
+
+#ifdef _WIN32
+#define EB_SCANF sscanf_s
```

Comment:
sscanf should be avoid for security

Reply:
Hi tianjunwork,
Thanks for the info. I have resolved it in commit (ac04dde7b9d46aab422a093d2cb4070379b32e54) in the PR